runtime/logging.go

```diff
@@ -40,7 +40,7 @@ func NewLoggingHandler(inner http.Handler) http.Handler {
 func (h *LoggingHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	requestID := atomic.AddUint64(&h.requestID, uint64(1))
 
-	recorder := newRecorder(w, r, requestID, loggingEnabled(logrus.DebugLevel))
+	recorder := newRecorder(w, r, requestID, loggingEnabled(logrus.InfoLevel))
```

Comment:
Hrmmm. This means that we're going to buffer all responses whether or not we end up logging them. I'm worried about the cost of the copy. I'd like to know that this isn't going to impact performance significantly.

Reply:
Is there a performance suite for OPA that I can run?  Would make test before/after give us the numbers?

Reply:
We have a number of benchmarks in OPA. We don't have any that exercise the server like this. I think we need to send a bunch of requests to the server and measure the difference before & after this change. It would be useful to have two traffic profiles:

1. Small decision - e.g., true/false as is the case in typical API authorization use cases
2. Large decision - e.g., some big/nested JSON object as is the case in typical audit use cases